main/lua-xml/APKBUILD

```diff
@@ -17,28 +17,28 @@ for _v in $_luaversions; do
 	subpackages="$subpackages lua$_v-xml:split_${_v/./_}"
 done
 
-source="http://viremo.eludi.net/LuaXML/LuaXML_$pkgver.zip
-0001-Fix-error-where-TAG-should-be-xml.TAG.patch"
-
-_builddir="$srcdir"
+source="$pkgname-$pkgver.tar.gz::https://github.com/LuaDist/luaxml/archive/$pkgver.tar.gz
+	0001-Fix-error-where-TAG-should-be-xml.TAG.patch"
 
+builddir="$srcdir"
 prepare() {
-	cd "$_builddir"
+	cd "$builddir"
 	for i in $source; do
 		case $i in
 		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
 		esac
 	done
 
+	dest="$pkgname-$pkgver"
```

Comment:
Custom variables inside functions should be always declared as `local`, if not needed in more functions. Moreover, this is totally unnecessary, just use `buildddir` properly.